runners/console/index.ts

```diff
@@ -79,6 +79,26 @@ export class Console extends Runner {
         return result;
     }
 
+    runCreateFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let workspaceDir = path.join(this.getWorkingDirectory(), "devonfw", "workspaces", "main");
```

Comment:
Path of the file to be created (relative path to the workspace directory)
should be (relative path to  /root). We would have to change it here, too. 
 @GuentherJulian